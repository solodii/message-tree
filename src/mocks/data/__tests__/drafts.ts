import {
  drafts,
  draftsBySubject,
  invalidDraft,
  validDraft
} from '../drafts';

test('drafts', () => {
  expect(drafts).toMatchSnapshot();
});

test('draftsBySubject', () => {
  expect(draftsBySubject).toMatchSnapshot();
});

test('invalidDraft', () => {
  expect(invalidDraft).toMatchSnapshot();
});

test('validDraft', () => {
  expect(validDraft).toMatchSnapshot();
});
