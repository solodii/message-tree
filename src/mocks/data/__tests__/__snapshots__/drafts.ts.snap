// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`drafts 1`] = `
Array [
  Object {
    "subjectId": "",
    "text": "",
  },
  Object {
    "subjectId": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
    "text": "draft for 0a6a7bfd-a212-4ae1-8125-6c7486213a94",
  },
]
`;

exports[`draftsBySubject 1`] = `
Object {
  "": Object {
    "subjectId": "",
    "text": "",
  },
  "0a6a7bfd-a212-4ae1-8125-6c7486213a94": Object {
    "subjectId": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
    "text": "draft for 0a6a7bfd-a212-4ae1-8125-6c7486213a94",
  },
}
`;

exports[`invalidDraft 1`] = `
Object {
  "subjectId": "",
  "text": "",
}
`;

exports[`validDraft 1`] = `
Object {
  "subjectId": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
  "text": "draft for 0a6a7bfd-a212-4ae1-8125-6c7486213a94",
}
`;
